### PR TITLE
Clean up CloudBigtableOptions documentation.

### PR DESCRIPTION
**Pull request by mgarolera**

### CODE REVIEW

File: bigtable-hbase-dataflow/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Here's [an example](https://github.com/GoogleCloudPlatform/cloud-bigtable-client
 
 To run the example via maven, use the following command, with the values in the brackets replaced with your information:
 
-`mvn exec:java -Dexec.mainClass="com.google.cloud.bigtable.dataflow.example.HelloWorldWrite" -Dexec.args="--runner=BlockingDataflowPipelineRunner --project=[some_project] --stagingLocation=gs://[some_bucket] --bigtableProject=[some_project] --bigtableClusterId=[cluster_name] --bigtableZone=[some_zone] --bigtableTable=[someTableName]" -Dexec.classpathScope="test"`
+`mvn exec:java -Dexec.mainClass="com.google.cloud.bigtable.dataflow.example.HelloWorldWrite" -Dexec.args="--runner=BlockingDataflowPipelineRunner --project=[some_project] --stagingLocation=gs://[some_bucket] --bigtableProjectId=[project_id] --bigtableClusterId=[cluster_id] --bigtableZoneId=[zone_id] --bigtableTableId=[table_id]" -Dexec.classpathScope="test"`
```
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/README.md]: some_project -> project_id ?

**Comment by kevinsi4508** [comment]
Are these 3 files part of this PR? READ.md, HelloWorldWrite.java and SourceRowCount.java?

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableOptions.java [hunk 2]
```diff
@@ -21,29 +21,29 @@
 
 /**
  * CloudBigtableOptions is an extension of {@link DataflowPipelineOptions} containing the
- * information required for a connection to Cloud Bigtable.
+ * options to configure a Dataflow pipeline that uses Cloud Bigtable.
  */
-@Description("Options used to configure CloudBigtable.  " +
-   "see https://cloud.google.com/bigtable/ for more information.  " +
-   "See https://cloud.google.com/bigtable/docs/creating-cluster for getting started with Bigtable.")
+@Description("Options used to configure a Cloud Bigtable Dataflow pipeline. " +
```
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableOptions.java]: Options used to configure a Dataflow pipeline that uses Cloud Bigtable. ?

File: bigtable-hbase-dataflow/README.md (hunk 1 above)
Comment (mgarolera) [inline bigtable-hbase-dataflow/README.md]: Done.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableOptions.java (hunk 2 above)
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableOptions.java]: Done.

**Comment by kevinsi4508** [comment]
LGTM (wait for Solomon's and Jeff's review)

**Comment by sduskis** [comment]
Cloud Bigtable uses ID to identify the specific portion of a full name of an object (ie. /project/{projectId}/zone/{zoneId}/cluster/{clusterId}/table/{tableId}.  The full path is a name in Cloud Bigtable terminology.

**Comment by sduskis** [comment]
Oh, I see.  We were inconsistent between our configuration class and our documentation.  Our Configuration classes use Id.  I'm good with this.  LGTM.

**Comment by hegemonic** [comment]
LGTM

**Pull request merged**